pkg/charts/parse.go

```diff
@@ -79,6 +80,19 @@ func GetPackage(rootFs billy.Filesystem, name string) (*Package, error) {
 	if err != nil {
 		return nil, err
 	}
+	// version and packageVersion can not exist at the same time although both are optional
+	if packageOpt.Version != nil && packageOpt.PackageVersion != nil {
+		return nil, fmt.Errorf("Cannot have both version and packageVersion at the same time")
+	}
+	var version *semver.Version
+	if packageOpt.Version != nil {
+		temp, err := semver.Make(*packageOpt.Version)
```

Comment:
Might run this through Matt's validator as well: https://github.com/mattfarina/semver-isvalid/blob/main/pkg/semver/valid.go

Reply:
`semver.Make` function also does the validation as a part of the making process. 
I skim through both implementations and do not see many differences in the items/rules they check.  

We can use Matt's validator for sure if we want.